pkg/secrethub/secret_version.go

```diff
@@ -115,7 +117,13 @@ func (s secretVersionService) GetWithData(path string) (*api.SecretVersion, erro
 		return nil, errio.Error(err)
 	}
 
-	return s.get(secretPath, true)
+	secretVersion, err := s.get(secretPath, true)
+	if api.IsErrNotFound(err) {
+		return nil, ErrSecretNotFound(path, err)
```

Comment:
We discussed this in person.
(Yes, this error should also be returned when not fetching data.
List already errors with the path of the secret included.)

Reply:
I thought we concluded we wanted to make `list` return the same error as `get` for consistency.

Reply:
What do you think?

Reply:
That's right, I forgot to mention it, but I think it should be done in a separate PR.

Reply:
Why? They're super related changes, right?

Reply:
I would want to release them both in the same release, therefore also merge them in the same PR.

Reply:
Yes, they are related changes and it would be nice to have them both, but I think this feature has a higher priority than modifying the error returned by the list command for consistency, and the latter requires a couple more design decisions that we might need to discuss.

Reply:
Oke, if you think it will require a lot extra time, we can postpone it a bit for now. Can you create a ticket to make sure this will eventually be picked up?

Reply:
Yes, good point :+1: